mslice/scripting/__init__.py

```diff
@@ -0,0 +1,13 @@
+from mantid.simpleapi import GeneratePythonScript
+from mslice.models.workspacemanager.workspace_provider import get_workspace_handle
+from mslice.util.qt import QtWidgets
+
+
+def generate_script(plot_window, ws_name, filename):
+    if filename is None:
+        path = QtWidgets.QFileDialog.getSaveFileName(plot_window, 'Save File', '', 'Python (.py)')
+        if path is u'':
+            raise RuntimeError("save_file_dialog cancelled")
```

Comment:
I don't think we should raise an exception in this case. If the user cancelled the dialog then we should just return and let them carry on.